draft/schemas/trailer.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$id": "https://w3id.org/kim/amb/draft/schemas/trailer.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Trailer",
+    "description": "Ein Trailer/Video-Teaser zu einer Ressource",
+    "type": "object",
+    "properties": {
+        "type": {
+            "title": "Type",
+            "type": "string",
+            "enum": [
+                "VideoObject"
+            ]
+        },
+        "contentUrl": {
+            "title": "Content URL",
+            "description": "The download URL of the media object",
+            "type": "string",
+            "format": "uri"
+        },
+        "encodingFormat": {
+            "title": "Encoding Format",
+            "description": "The media type of the media object, conforming to https://www.iana.org/assignments/media-types/media-types.xml",
+            "type": "string"
```

Comment:
```suggestion
            "type": "string",
            "pattern": "\\w+\\/[\\w+]+",
```

Reply:
In https://www.iana.org/assignments/media-types/media-types.xml sind ja alle möglichen Werte aufgeführt. Ob man das nochmal durch ein Pattern absichert oder nicht ist mir eigentlich gleich. Jedenfalls, falls ein Pattern genutzt wird, so müssten im zweiten Teil noch ein paar mehr Sonderzeichen mit rein um zB `application/mpeg4-generic` oder `application/vnd.century-systems.tcp_stream` abzudecken. Soweit ich es überblicke, sollte `\\w+\\/[\\w+-._]+` alle Sonderzeichen abdecken.

Reply:
Diese Antwort auf Stack Overflow spricht sich gegen die Nutzung einer RegEx für IANA Meida Types aus: https://stackoverflow.com/questions/12663354/regular-expression-for-filetypes#12663541

[Wikidata](https://www.wikidata.org/wiki/Property:P1163#P1793) hingegen verwendet folgende RegEx: `(application|audio|example|font|image|message|model|multipart|text|video)/[a-zA-Z0-9\-+.]+`

Ich schlage vor, dass wir auch die Wikidata RegEx verwenden. Ok?

Reply:
Ok, sollte sich im Nachhinein herausstellen (was ich nicht glaube), dass in der Praxis doch noch weitere Typen verwendet werden, die hiermit nicht möglich sind, so kann dann immer noch nachgebessert werden.

Reply:
Ist es ein Problem, dass der Unterstrich `_` für zB `application/vnd.century-systems.tcp_stream` fehlt? Meiner Meinung nach werden die entsprechenden Werte vermutlich eh nirgends verwendet. Wollen wir ihn der Vollständigkeit halber dennoch aufnehmen?

Reply:
> Wollen wir ihn der Vollständigkeit halber dennoch aufnehmen?

Erledigt in [c9cfeeb](https://github.com/dini-ag-kim/amb/pull/158/commits/c9cfeeb1ae3d81bb25840fe2c3da80782d196528).